pkg/app/config.go

```diff
@@ -103,3 +147,24 @@ func validateConfig(c Config) error {
 	}
 	return nil
 }
+
+// Returns the data inside of a given secret
+func getSecretData(secretName, namespace string) (map[string][]byte, error) {
```

Comment:
I thought we already had a method to get secrets. I seem to recall a secrets.go that @fabianvf added in a previous patch. Seems that would be useful. If not this seems useful in general to me.

Reply:
Yeah we did have a method but it was being used to get secrets associated with a specific APB. It also required me to initialize a secrets cache in order to use those methods. I'm not against making this globally available and putting it in a utils package.